R/get_season_team_stats.R

```diff
@@ -62,6 +62,17 @@ fb_season_team_stats <- function(country, gender, season_end_year, tier, stat_ty
 
     season_stats_page <- .load_page(season_url)
 
+    # have included this to differentiate between how different leagues handle ladders/tables
+    league_tables <- season_stats_page %>% rvest::html_nodes("#content .table_wrapper") %>% rvest::html_elements("h2") %>% rvest::html_text()
+
+    # we either want to detect the presence of League Table or Regular season and get the index of that to be able to extractg the table we want
+    if(length(grep("league table", tolower(league_tables))) > 0) {
+      league_tables_idx <- grep("league table", tolower(league_tables))
+    } else {
+      league_tables_idx <- grep("^Regular season", league_tables)
+    }
```

Comment:
The `h2` headings are the key here that we can try to use, but for some leagues tables are called 'League Table' while others are called  `Regular season`. The Mexican League does make things even harder by having 'Clausura 2020 Regular Season', etc, but hopefully the regex in here will handle these.